cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -748,7 +748,16 @@ func removeOldUnregisteredNodes(unregisteredNodes []clusterstate.UnregisteredNod
 				klog.Warningf("Failed to remove node %s: node group min size reached, skipping unregistered node removal", unregisteredNode.Node.Name)
 				continue
 			}
-			err = nodeGroup.DeleteNodes([]*apiv1.Node{unregisteredNode.Node})
+
+			if context.AutoscalingOptions.ExternalNodeDeletion {
+				// allow the cloud provider to handle node deletion instead of deleting nodes through downscaling replica counts
+				fmt.Printf("MarkNodesForDeletion (removeOldUnregisteredNodes)\n")
```

Comment:
i think we'll need to upgrade these printf's to log messages, perhaps using `V(2).InfoS` level.